packages/core/src/components/DateRangePicker/types.ts

```diff
@@ -0,0 +1,56 @@
+import { Dispatch, SetStateAction } from 'react';
+import { DateTime } from 'luxon';
+import { DatePickerProps } from '@components/DatePicker/types';
+import { FieldContextValue } from '@components/Field/FieldProvider';
+
+export type LastFocusedElement = 'from' | 'to';
+
+export type DateRangePickerProps = Omit<
+  DatePickerProps,
+  'isOpenToggle' | 'value' | 'defaultValue' | 'onChange'
+> & {
+  value?: [string | undefined, string | undefined]; // depends on the format
+  defaultValue?: [string, string]; // depends on the format
+  status?: FieldContextValue['status'];
+  onChange?: (dates?: [Date | null, Date | null]) => void;
+  onOpen?: () => void;
+  onClose?: () => void;
+  onError?: (
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    date: any,
+    error?: string | null,
+  ) => void;
```

Comment:
It looks like unknown can be used here

```suggestion
  onError?: (date: unknown, error?: string | null) => void;
```

Reply:
Agree, will be changed